src/components/SingleProduct.js

```diff
@@ -29,7 +29,7 @@ function SingleProduct({ product }) {
               : ''}
           </p>
           <p>
-            Cost: {product.cost} Golds
+            Cost: {product.cost} Gold
             <br />
```

Comment:
I intentionally misspelled golds here, but I'll take the correctness.

Reply:
Haha, I am a fan of the whimsical spellings